business/workloads.go

```diff
@@ -1081,7 +1082,8 @@ func fetchWorkload(layer *Layer, namespace string, workloadName string, workload
 		// Check if namespace is cached
 		// Namespace access is checked in the upper call
 		if IsNamespaceCached(namespace) {
-			repset, err = kialiCache.GetReplicaSets(namespace)
+			// repset, err = kialiCache.GetReplicaSets(namespace)
```

Comment:
I'll review it in detail, but disabling the cache for this doesn't look correct.
Note that most of the tools today use the informers (cache) as a source of truth for the controllers.

But I'll fetch the whole patch to add detailed comments, but this has created a warning on this PR.

Reply:
Oh, that's totall wrong, that was for debugging.  I'll fix it, sorry.

Reply:
pushed change, stupid of me